challenge.js

```diff
@@ -43,6 +52,25 @@
 
 // ANSWER
 
+<body class='a'>
+  <div class='b'>
+  <div class='c'>
+  <div class='d'>
+    <p class='e'>
+
+Selector:
+
+Specific class selector:
+.e {
+
+}
+
+Or you could call a less specific element selector since it is the only <p>
+
+p {
+
+}
+
```

Comment:
Both reasonable, given how the question is worded, so no points off. However, the answer that we were looking for was closer to `body > div > p`.